command/operator_migrate.go

```diff
@@ -366,11 +378,16 @@ func dfsScan(ctx context.Context, source physical.Backend, cb func(ctx context.C
 				}
 			}
 		} else {
-			err := cb(ctx, key)
-			if err != nil {
-				return err
-			}
-
+			// Pooling
+			permitPool.Acquire()
+			go func() error {
+				defer permitPool.Release()
+				err := cb(ctx, key)
+				if err != nil {
+					return err
```

Comment:
I think this error doesn't make it to the outside `dfsScan ` function. Something should be done with the error.

Reply:
What would you suggest? Sending via channel?

Reply:
That could work, or using an ErrorGroup should work I think. I can't think of any particular standardization at the moment that I'm particularly partial toward, just that there should be some error handling here. I can circle back with some other folks to see if there's a particular preference or standardization we have as well.